### DISCUSSION TITLE
Ultima latest version theme files issue

### DISCUSSION THREAD

**Discussion by kubila** [primefaces/community#1071] [PrimeVue Templates] [answered]
Recently we've bought your Ultima template. But there is a problem, when we follow instructions on the Readme, the browser can't find theme css file. When we look for theme files we've realized that only the scss files are provided, but we need theme css files. Also the index.html file referring to a css file that does not exist. Where can we get theme files? We only tried versions of 5.2.0 and 5.0.0.

**Comment by buraksaglam089** [comment]
Hello,

Did you compile the sass with command before you run the project?

Regards.

**Reply by kubila** [reply, marked as answer]
@buraksaglam089 Thanks for the reply.

Found the command [here](https://github.com/orgs/primefaces/discussions/236?sort=old); didn't work without 'npx' prefix though.

In case anyone trying to find a solution; run the command below via npx at the root of your project, after running this command theme css files will be compiled and created.

`npx sass --update public/layout/:public/layout/ --no-source-map`

I think all those steps should be **_DOCUMENTED_**. There's nothing in the Readme.md file in your latest version of Ultima template regarding this issue.

**Reply by patsuhiko** [reply]
Thank you @kubila 
I had the same problem and solved it with your answer. Also from my side: the line should be documented.
